hack/validate-kustomize.sh

```
@@ -12,7 +12,7 @@ normal="$(tput sgr0)"
 
 for kustomization in "$BASEDIR"/../config/*/**/kustomization.yaml; do
   path="$(dirname "$kustomization")"
-  dir="$(realpath --relative-to "$BASEDIR"/.. "$path")"
+  dir="${path#$BASEDIR/../}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)